src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -61,7 +61,7 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
   const { data, error, isLoading } = useAxiosGet(url);
   useEffect(() => {
     if (data) {
-      ingestLines(data.split("\n"), logType);
+      ingestLines(data.trim().split("\n"), logType);
```

Comment:
trim removes spaces from both ends of the string. I fear this may result in us removing intentional blank lines if there are some at the head or tail of a log file.

Reply:
I swapped it to `trimEnd` instead, which I think is good enough for now.

The new line at the end of any file doesn't render in the virtualized list for some reason, so it needs to be chopped off